tests/e2e/map/map_spatial.spec.js

```diff
@@ -31,764 +52,1767 @@ test.describe.skip('Map: Spatial interactions', () => {
+
+          // Updates the URL
+          await expect(page).toHaveURL(/search\?sp\[0\]=-28\.\d+%2C-9\.\d+/)
+
+          // Populates the spatial display field
+          await expect(page.getByTestId('spatial-display_point')).toHaveValue(/-9\.\d+,-28\.\d+/)
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
+          await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-point-spatial-drawn.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Select the point spatial type
-        await page.getByRole('button', { name: 'spatial-selection-dropdown' }).click()
-        await page.getByRole('button', { name: 'Point' }).click()
+      test.describe('When typing a new point in the spatial display', () => {
+        test('renders correctly', async ({ page }) => {
+          await interceptUnauthenticatedCollections({
+            page,
+            body: commonBody,
+            headers: commonHeaders,
+            additionalRequests: [{
+              body: pointBody,
+              headers: {
+                ...commonHeaders,
+                'cmr-hits': '2'
+              },
+              paramCheck: (parsedQuery) => parsedQuery?.point?.[0]?.match(/42\.\d+,4\.\d+/)
+            }]
+          })
+
+          await page.goto('/')
+
+          // Wait for the map to load
+          await page.waitForSelector('.edsc-map-base-layer')
+
+          // Select the point spatial type
+          await page.getByRole('button', { name: 'spatial-selection-dropdown' }).click()
+          await page.getByRole('button', { name: 'Point' }).click()
+
+          // Enter the spatial point
+          await page.getByTestId('spatial-display_point').focus()
+          await page.keyboard.type('4.5297,42.1875')
+          await page.keyboard.up('Enter')
+
+          // Updates the URL
+          await expect(page).toHaveURL('search?sp[0]=42.1875%2C4.5297&lat=4.5297&long=42.1875&zoom=21')
+
+          // Populates the spatial display field
+          await expect(page.getByTestId('spatial-display_point')).toHaveValue('4.5297,42.1875')
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
+          await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+          // Wait for the map to update
+          await page.waitForTimeout(250)
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-point-spatial-typed.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Add the point to the map
-        await page.mouse.click(1000, 500)
+      test.describe.skip('When editing a point', () => {
```

Comment:
```suggestion
      test.describe('When editing a point', () => {
```

Reply:
We decided not to edit spatial right now, leaving these as skipped